### DISCUSSION TITLE
font weight not applied

### DISCUSSION THREAD

**Discussion by shelcia** [vercel/next.js#45497] [Help]
I am new to NextJS, I want to include local fonts like this as given in (https://nextjs.org/docs/basic-features/font-optimization)
But apart from 400 font weight nothing else loads. I had inspected the element, it has font size 700 but was not able to utilise right font weight on UI.

```jsx
import localFont from "@next/font/local";

export const satoshi = localFont({
  src: [
    {
      path: "../assets/fonts/Satoshi-Regular.woff",
      weight: "400",
      style: "normal",
    },
    {
      path: "../assets/fonts/Satoshi-Medium.woff",
      weight: "500",
      style: "normal",
    },
    {
      path: "../assets/fonts/Satoshi-Bold.woff",
      weight: "600",
      style: "normal",
    },
    {
      path: "../assets/fonts/Satoshi-Black.woff",
      weight: "700",
      style: "normal",
    },
  ],
});
```

**Comment by ciruz** [comment]
Hey,

maybe there is something wrong with this particular font, did you try it with a total different font too?

**Comment by lazim-mv** [comment]
I have the same issue i have tried with 2 fonts still its same so instead of using next/font i have used google fonts import tag.
